src/dict.c

```diff
@@ -48,6 +49,10 @@
 #include "serverassert.h"
 #include "monotonic.h"
 
+#ifndef static_assert
+#define static_assert(expr, lit) extern char __static_assert_failure[(expr) ? 1 : -1]
```

Comment:
We moved to c11, so we just have static asserts now:
```suggestion
#define static_assert(expr, lit) _Static_assert(expr, lit)
```